test/big.test.js

```diff
@@ -1,112 +0,0 @@
-'use strict'
-
-const test = require('tap').test
-const FormData = require('form-data')
-const Fastify = require('fastify')
-const multipart = require('..')
-const http = require('http')
-const stream = require('readable-stream')
-const Readable = stream.Readable
-const Writable = stream.Writable
-const pump = stream.pipeline
-const eos = stream.finished
-const crypto = require('crypto')
-
-// skipping on Github Actions because it takes too long
-test('should upload a big file in constant memory', { skip: process.env.CI }, function (t) {
```

Comment:
This test is actually important to check if we do not cause a memory leak.

Reply:
Don't worry I just remove it for a clean start. I will add this when we will agree on the implementation.